### DISCUSSION TITLE
比 GOOGLE MAP更真實! -Immersive View

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#594] [General]
Google近期在開發者大會上展示未來會推出的新科技，其中有項「Immersive View」沈浸觀看模式，結合了Google earth的街景模式以及「電腦視覺」(computer vision) 技術。這項沉浸觀看技術，反映出不同時間內街道、店家的改變，使用者還可以進入店家，觀看內部的裝潢等等。跟Google earth有幾分相似。期待它的上市以及正式的使用!

文章來源:https://buzzorange.com/techorange/2022/05/13/google-immersive-view/

**Comment by handsomeniko** [comment]
推 謝謝分享

**Comment by sys511613** [comment]
有了這個技術感覺在家就可以去各個地方參觀
而且有開放 ARCore給第三方廠商，這樣未來的遊戲和很多應用都會變得更方便好玩！

**Comment by Sazmafuji** [comment]
這已經算metaverse了吧

**Comment by cordydeng** [comment]
太厲害了吧，有如身歷其境ㄟ

**Comment by jcpan821** [comment]
以後在家也能逛街了

**Comment by awkward-willy** [comment]
這樣的技術能讓大家在家就能有環遊世界的感覺
在這個疫情的時代一定能大放異彩

**Comment by ghost** [comment]
宅男宅女們有福啦>///<

**Comment by Lai27330** [comment]
謝謝分享，好高級的科技感，而且可以排出一些外表好看但內裝不符的店

**Reply by Rosancl** [reply]
又抓到你打錯字ㄌ

**Comment by alexthebigpp** [comment]
整個地球的地圖已經夠複雜了，還要做成3D感覺就要很龐大的運算能力

**Comment by Rosancl** [comment]
真的開始有越來越多沉浸式科技的應用出現

**Comment by stan890425** [comment]
這樣就不用出門逛街了~~

**Comment by liangyinglly** [comment]
平常使用Google earth的街景模式時總覺得它跑很慢，很不好用，希望Google除了推出新科技，也能改善使用不變的問題。

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by HaoFish123** [comment]
感謝分享，好想趕快用用看。

**Comment by Jimmy911218** [comment]
這技術能用在旅遊上一定能提供遊客很大的幫助
平常查旅遊景點時只能看別人拍過的可能不準確會有修圖等等
但這能比較客觀的呈現很棒

**Comment by treeann** [comment]
當初google map推出的時候真的覺得很厲害

**Comment by jason79461385** [comment]
不知道台灣什麼時候能有 希望哪天可以體驗到

**Comment by 110409538** [comment]
好酷,如果可以結合運動的話感覺不錯,比如跑步路線記錄的呈現方式.

**Comment by etin2468** [comment]
真的能在家環遊世界ㄟ，好酷

**Comment by Lynn12378** [comment]
好酷喔!感覺這樣以後可以更好的去認路了不熟悉的路了
